docs/basic/none.md

```diff
@@ -54,6 +54,26 @@ Bisa dilihat pada program di atas output tiap statement adalah berbeda-beda sesu
 >
 > Pembahasan detail mengenai special name ada di chapter [Special Names ➜ Attribute __name__ milik class type](/basic/special-names#a294-attribute-__name__-milik-class-type)
 
+## A.31.2. Penggunaan `is`
```

Comment:
I removed this section because `is` keyword is already explained in this section https://dasarpemrogramanpython.novalagung.com/basic/object-id-reference#a213-operasi-logika-via-keyword-is

Reply:
on second though, I think it is better to explain it but in a different direction